client/src/components/common/boost/BoostConfigurator.tsx

```diff
@@ -45,6 +79,16 @@ export default function BoostConfigurator({
                 {camelCaseToStartCase(config.type)}
```

Comment:
Can you make this text vertically centred? Looks a bit awkward now the button makes the accordion taller. You could also consider making the button slightly smaller with `size='sm'`, see what you think.

Reply:
Size small is alright on Laptops but maybe on a mobile device it would be hard to click especially because all the buttons are so close by. Do we intend to use dashboard on mobile devices or laptops/PCs or both?

Reply:
Good question! Dashboard will be used by both mobile and PCs.